.github/workflows/build-webapi.yaml

```diff
@@ -11,7 +11,7 @@ on:
 
 env:
   DOCKERFILE_PATH: "CarbonAware.WebApi/Dockerfile"
-  HEALTH_ENDPOINT: "0.0.0.0:8080/emissions/bylocation?location=eastus"
```

Comment:
suggestion: Keep this one too, so we know at least the endpoint works. Maybe creating a new step in the CI that runs the 'data' endpoint would be good to have.